### PR TITLE
added e-shadow and e-gradient

### PR DESCRIPTION
**Pull request by ankur-dwivedi**

### CODE REVIEW

File: lib/imagekitio/sdk/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module ImageKitIo
   module Sdk
-    VERSION = '2.3.0'
+    VERSION = '2.3.1'
```
Comment (imagekitio) [inline lib/imagekitio/sdk/version.rb:3]: Given we are removing a lot of transformation parameter, we need a major version bump.

File: .github/workflows/gempush.yml [hunk 2]
```diff
@@ -10,10 +10,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby 2.6
+    - name: Set up Ruby 3.0
```
Comment (imagekitio) [inline .github/workflows/gempush.yml:13]: What is the impact of this change on users in terms of ruby version compatibility?

File: README.md [hunk 3]
```diff
@@ -30,6 +31,13 @@ Table of contents -
 # Quick start guide
 Get started with [official quick start guide](https://docs.imagekit.io/getting-started/quickstart-guides/ruby-on-rails) for integrating ImageKit in Ruby on Rails.
 
+## Changelog
+### SDK Version 3.0.0
+#### Breaking changes
+**1. Overlay syntax update**
+* In version 3.0.0, we've removed the old overlay syntax parameters for transformations, such as `oi`, `ot`, `obg`, and [more](https://docs.imagekit.io/features/image-transformations/overlay). These parameters are deprecated and will start returning errors when used in URLs. Please migrate to the new Layers syntax that supports overlay nesting, provides better positional control, and allows more transformations at the layer level. You can start with [examples](https://docs.imagekit.io/features/image-transformations/overlay-using-layers#examples) to learn quickly.
```
Comment (imagekitio) [inline README.md:38]: Use small `l` instead of capital `L` in layers. Sab jagah same feedback.

File: .github/workflows/gempush.yml (hunk 2 above)
Comment (ankur-dwivedi) [reply .github/workflows/gempush.yml]: the above change is not needed now the issue is fixed by using `bundler 2.4.22` in ci  `Set up Ruby 2.6` was using a higher version of `bundler`.

File: README.md (hunk 3 above)
Comment (ankur-dwivedi) [reply README.md]: done

**Comment by ahnv** [comment]
The CI has been retained on Ruby Version 2.6.5, the reason CI was failing because `ruby/setup-ruby` was installing a newer version of bundler, so in this MR, the version of bundler is locked to the version that is supported by 2.6.5

**Review by ahnv — APPROVED** [review]

**Pull request merged**